.github/workflows/bbw_build_container.yml

```diff
@@ -156,6 +163,7 @@ jobs:
               --manifest ${{ env.REPO }}:${{ env.IMG }} \
               -f $GITHUB_WORKSPACE/Dockerfile \
               --build-arg base_image=${{ matrix.image }} \
+              --build-arg CLANG_VERSION=${{ matrix.clang_version }} \
```

Comment:
Upper case is probably good and improves visibility, it makes me wonder if we should not normalize it then for `base_image` and `mariadb_branch`?